common/static/common/js/spec/components/paging_spec.js

```diff
@@ -58,9 +58,7 @@ define([ "jquery", "js/common_helpers/ajax_helpers", "URI",
             initialize : function() {
                 this.registerSortableColumn('name-col', 'Name', 'name', 'asc');
                 this.registerSortableColumn('date-col', 'Date', 'date', 'desc');
-                this.registerFilterableColumn('js-asset-type-col', gettext('Type'), 'asset_type');
```

Comment:
It is a shame this code went into Studio without any tests. It would be better if we wrote a few tests to get coverage. I'll look into this now I've finished my reviews.